### PR TITLE
added code coverage support

### PR DESCRIPTION
**Pull request by harshkothari410**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "test-db": "npm run resetdb && mocha -R dot --recursive tests/db",
     "test-realtime": "mocha -R dot --recursive tests/realtime",
     "test-view": "NODE_ENV=test mocha -R dot --recursive --compilers js:babel-core/register --require ./tests/view/setup.js tests/view",
-    "test": "npm run test-db && npm run test-api && npm run test-view && npm run test-realtime && npm run test-token-req && npm run test-token-notreq && npm run test-disablehttp",
+    "test": "istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R dot --recursive tests/api tests/db && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage && npm run test-view && npm run test-realtime && npm run test-token-req && npm run test-token-notreq && npm run test-disablehttp",
```
Comment (iamigo) [inline package.json]: Are the tests which are executed from "test-view" and "test-realtime" and "test-token-req" counted towards the code coverage?

Comment (harshkothari410) [inline package.json]: I thought for code coverage will only cover db and API test. For test view I need to check because in needs some compilation.

Comment (iamigo) [inline package.json]: They're all just mocha tests

**Comment by harshkothari410** [comment]
Yeah I am finding a way to integrate view test thats why I am not able to update it. But yeah sure once I find that I will update it in different PR.

**Pull request merged**